tasks/failover_ip.yml

```diff
@@ -0,0 +1,21 @@
+- name: Add failover network interface alias
+  template:
+    src=network/interfaces.d/failover-ip
+    dest="{{ COMMON_FAILOVER_INTERFACE_CONFIG }}"
+    owner=root
+    group=root
+    mode=0644
+
+- name: Ensure failover network is included in configuration file
+  lineinfile:
+    dest=/etc/network/interfaces
+    state=present
+    line="source {{ COMMON_FAILOVER_INTERFACE_CONFIG }}"
```

Comment:
Looks like the `source-directory` directive was in [trusty](http://manpages.ubuntu.com/manpages/trusty/man5/interfaces.5.html), but was replaced by `source` in [precise](http://manpages.ubuntu.com/manpages/precise/man5/interfaces.5.html) and [xenial](http://manpages.ubuntu.com/manpages/xenial/man5/interfaces.5.html).

The default I've been seeing on the OVH VMs is `source /etc/network/interfaces.d/*.cfg`, though the dedicated servers had no `source` line.

On second thought, it's better to stick with the current standard.  Please see  366f6bd.